Makefile

```diff
@@ -138,3 +138,7 @@ reload-prod:
 
 stop-prod:
 	@kill -SIGTERM `cat $PROD_PID`
+
+.PHONY: environ
```

Comment:
It is NOT `.PHONY` target, since it shouldn't corrupt existing file if accidentally run.